features/fedcm.yml

```diff
@@ -0,0 +1,8 @@
+name: Federated credential management
+description: The `IdentityCredential` API. Also known as FedCM.
```

Comment:
```suggestion
description: The `IdentityCredential` API delegates authentication to a third-party identity provider directly, instead of depending on third-party cookie. Also known as FedCM.
```

Reply:
Just missing an "s" in "third-party cookies", which I'll add.